nilearn/datasets/func.py

```diff
@@ -2126,6 +2126,75 @@ def fetch_development_fmri(n_subjects=None, reduce_confounds=True,
                  description=fdescr)
 
 
+def fetch_surf_tva_localizer(data_dir=None, verbose=1, resume=True):
+    """Download the data from one subject of the InterTVA dataset,
+        for the event-related voice localizer run.
+        The data provided here is a set of beta maps each estimated
+        for one of the 144 audio stimuli presented to the subject,
+        and projected onto the cortical surface in the fsaverage5
+        template space, for the left hemisphere.
+        It is used for the example of the cortex-based searchlight decoding.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        Path of the data directory. Used to force data storage in a specified
+        location. Default: None
+
+    verbose: int, optional (default 1)
+        Defines the level of verbosity of the output.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, the interest attributes are :
+         - 'func_left': Paths to Gifti files containing resting state
+                        time series left hemisphere
+         - 'phenotypic': array containing tuple with subject ID, age,
+                         dominant hand and sex for each subject.
+         - 'description': data description of the release and references.
+
+    References
+    ----------
+    :Download: https://openneuro.org/datasets/ds001771/
+
+    """
+
+    # Preliminary checks and declarations
+    dataset_name = 'intertva_localizer_surface'
+    intertva_dir = _get_dataset_dir(dataset_name, data_dir=data_dir)
+    dataset_dir = _get_dataset_dir('InterTVA_sub-41_surfdata', data_dir=intertva_dir)
+    url = 'https://osf.io/aekp7/download'
+
+    # Dataset description
+    fdescr = _get_dataset_descr(dataset_name)
+
+    # First, get the metadata, i.e the labels to be used for the searchlight decoding
+    label_path = 'labels_voicelocalizer_voice_vs_nonvoice.tsv'
+    label_file = (label_path, url , {'uncompress': True})
+
+    label_localpath = _fetch_files(dataset_dir, [label_file], resume=resume,verbose=verbose)[0]
+    labels = np.genfromtxt(label_localpath, dtype='str', skip_header=1)[:,0]
+
+    # Secondly, get the mask of a region of interest for this task
+    mask_path = 'fsaverage5.lh.stg_sts.gii'
+    mask_file = (mask_path, url, {'uncompress': True})
+    mask = _fetch_files(dataset_dir, [mask_file], resume=resume, verbose=verbose)[0]
+
+    # Finally, get the data itself, i.e the 144 single-trial beta maps projected on the surface
+    gifti_file_list = []
+    for current_trial in range(144):
+        gifti_path = 'fsaverage5.lh.beta_{:04d}.gii'.format(current_trial+1)
+        gifti_file = (gifti_path, url, {'uncompress': True})
```

Comment:
Replace the name `gifti_file` with something like `download_info_gifti_`, since this is not in-fact a gifti-file.